.circleci/config.yml

```diff
@@ -54,14 +55,24 @@ jobs:
       - run:
           name: Install dependencies
           command: |
-            pip install -U virtualenv --user
+            python -m pip install --progress-bar off -U virtualenv --user
             mkdir -p ./venv
-            python -m virtualenv ./venv
+            python -m virtualenv ./venv --clear
             . venv/bin/activate
-            python -m pip install --progress-bar off ${SIGNAC_VERSION}
-            python -m pip install --progress-bar off -r requirements.txt
-            python -m pip install --progress-bar off -r requirements-test.txt
+            python -m pip install --progress-bar off -U pip>=20.3
             python -m pip install --progress-bar off -U codecov
+            python -m pip install --progress-bar off -U ${SIGNAC_VERSION}
+            # DEPENDENCIES can be "OLDEST", or "NEWEST"
+            if [ "${DEPENDENCIES}" == "OLDEST" ]; then
+              python -m pip install --progress-bar off \
+                jinja2==2.10 \
+                cloudpickle==1.1.1 \
+                deprecation==2 \
+                tqdm==4.43.0
```

Comment:
Would it be best to just put these dependencies in another file, similar to `requirements.txt`, and `requriements-test.txt`?

Reply:
This is a nice idea, and I think making a file called `requirements-oldest.txt` should do the work. But we'd then have to configure `depandabot` accordingly because I think it modifies all the files having the requirements keyword in it. (Which would make the implementation more complicated). 

@tommy-waltmann let me know what you think?

Reply:
I'm not sure how re-configuring `dependabot` would work, I think @bdice or @csadorf would be better able to comment on that.

Reply:
Right, as @kidrahahjo noted, the challenge is that we don't want dependabot to update the "oldest" dependency file. We might be able to add an exception but I couldn't figure out how to do that from the dependabot documentation. If someone else can identify a solution for that, I would definitely prefer that to hard-coding the "oldest" requirements in the CI config.

Reply:
If `dependabot` just looks for files that have the `requirements` substring in them, we could just have a file with a different name.

Reply:
I believe that dependabot looks in all files named `*requirements*.txt` and also `requirements/*.txt`, so we could name it `.circleci/ci-oldest-reqs.txt`?

Reply:
This is now done!